docs/wiki/The context - Logical layers and personas.md

```diff
@@ -0,0 +1,112 @@
+This persona will need to understand the required and optional configuration data (parameters) and will need to understand the architecture and "opinion" behind the solution.
+
+This persona isn’t consuming CARML directly, but is consuming CARML via a solution that relies on it:
+
+- Examples:
+  - Existing CARML solutions (These links provide examples of real world solutions using CARML):
+    - [AKS Landing Zone Accelerator](https://github.com/Azure/AKS-Landing-Zone-Accelerator)
+    - [AVD Landing Zone Accelerator](https://github.com/Azure/avdaccelerator)
+  - An internally developed multi-module solution for internal application or workload pattern
+
+> **Note**: Solution Consumers are not to be confused with end users. Solution Consumers are cloud administrators, deploying a complex solution and administering an application installed into such a cloud environment. An end user is someone who will use the  application or workload hosted in the cloud for his/her work (ex., information worker or application developer).
+
+### Typical Activities of a Solution Consumer
+
+- Clone or reference a multi-module solution (ex., AKS Landing Zone Accelerator).
+- Understand the reference architecture, key decisions.
+- Update parameters to include organization and environment specific data.
+- Deploy the template(s) of the multi-module solution.
+
+### How does a Solution Consumer benefit from CARML?
+
+- Start with known good reference implementations.
+- Consume Microsoft and organization-specific guidance.
+- Improve time to value for workload administrators and users.
+
+## Solution Developer
+
+![SolutionDevelopers](./media/Personas/SolutionDevelopers.png)
+
+A **Solution Developer** is not a user. Their focus is on using CARML to **build end-to-end solutions** that are opinionated. These solutions will be published (publicly and/or privately) for consumption by others and will be used many times by many organizations/teams, in many environments/subscriptions/landing zones).
+A Solution Developer uses the CARML library (its public or organization/customer specific implementation), with a chosen preference for using its modules via Bicep registry (public, private), or template specs, or direct reference or using Git submodules, etc. Solution Developers might benefit the most from overcoming the learning curve – because the standardization and scale they can achieve with CARML.
+
+- Modules: Use an existing code base to develop and test multi-module solutions (ex., a Landing Zone Accelerator, or an internal platform/workload team). This might also lead to creating/updating modules.
+- Continuous Integration: Use the provided CI environment to improve multi-module development and testing. This might also lead to improving CI.
```

Comment:
Do we mean that the CARML CI environment could be leveraged to test and validate solutions instead of modules? Or are we referring to a dedicated CI environment leveraged by the Solution Developer? In the first case, that can be challenging to achieve/convert as the CARML CI environment is quite modules specific.